app/javascript/shared/helpers/MessageFormatter.js

```diff
@@ -13,8 +13,9 @@ const TWITTER_HASH_REPLACEMENT =
 const USER_MENTIONS_REGEX = /mention:\/\/(user|team)\/(\d+)\/(.+)/gm;
 
 class MessageFormatter {
-  constructor(message, isATweet = false) {
+  constructor(message, isAPrivateNote = false, isATweet = false) {
```

Comment:
<img width="493" alt="Screenshot 2022-03-29 at 7 02 02 PM" src="https://user-images.githubusercontent.com/2246121/160622903-ec1f3488-94c3-4d23-9d9f-7e4340926657.png">



@iamsivin Let us keep the new attribute to last. Keep the existing so that the rest of the implementation is not affected.

Reply:
Done